doc/user/USAGE.md

```diff
@@ -69,13 +116,22 @@ Here is the list of plugins provided officially by F3D:
 
 > Note: If you downloaded the binaries from the Release page, it's not necessary to specify manually the plugins above, all of them are loaded automatically.
 
-Here is how the plugins are searched (by precedence order):
-1. Search the static plugins
-2. Consider the option given is a full path
-3. Search in the paths specified in `F3D_PLUGINS_PATH` environment variable
-4. Search in a directory relative to the F3D application: `../lib`
-5. Rely on OS specific paths (e.g. `LD_LIBRARY_PATH` on Linux or `DYLD_LIBRARY_PATH` on macOS)
+Here is how the plugins are searched (in preceding order):
+
+1. Search the static plugins.
+2. Consider the option given is a full path.
+3. Search in the paths specified in `F3D_PLUGINS_PATH` environment variable.
+4. Search in a directory relative to the F3D application: `../lib`.
+5. Rely on OS specific paths (e.g. `LD_LIBRARY_PATH` on Linux or `DYLD_LIBRARY_PATH` on macOS).
 
 You can also try plugins maintained by the community. If you have created a plugin and would like it to be listed here, please submit a pull request.
 
 - **Abaqus**: ODB support by @YangShen398 ([repository](https://github.com/YangShen398/F3D-ODB-Reader-Plugin))
+
+## Further Reading
+
+* [A list of all F3D command-line options.](OPTIONS.md)
+* [How to interact with scenes in F3D.](INTERACTIONS.MD)
```

Comment:
I feel like a quick introduction to interactions should be present in this guide, wdyt ?

Reply:
i agree, will add this in following commits.